lib/diagnostic.js

```diff
@@ -39,21 +47,31 @@ let owner = null;
 // Methods: vocalize
 
 // Replace null with your answer
-let pet = null;
+let pet = {
+  name: 'lucky',
+  species:'cat',
+  breed:'bobcat',
+  vocalization: 'meow'
+};
```

Comment:
Nice, but you forgot to add a `vocalize` method to this object. Something like this: `vocalize: function() {return this.vocalization}